src/content/docs/en/guides/upgrade-to/v5.mdx

```diff
@@ -662,6 +662,66 @@ Note that `src/env.d.ts` is only necessary if you have added custom configuratio
 
 <ReadMore>Read more about [TypeScript configuration in Astro](/en/guides/typescript/#setup).</ReadMore>
 
+### Changed: Actions submitted by HTML forms no longer use cookies
+
+In Astro 4.x, actions called from an HTML form would trigger a redirect with the result forwarded using cookies. This caused issues for large form errors and return values that exceeded the 4 KB limit of cookie-based storage.
+
+Astro 5.0 now renders the result of an action as a POST result without any forwarding. This reintroduces the "confirm form resubmission?" dialog when a user attempts to refresh the page, and will not persist action results across sessions.
```

Comment:
```suggestion
Astro 5.0 now renders the result of an action as a POST result without any forwarding. This introduces the "confirm form resubmission?" dialog when a user attempts to refresh the page, and will not persist action results across sessions.
```
Did we ever have the dialog in the first place? If not, we're not "re" introducing it.